src/main/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerapi/model/PrisonerDetailDto.kt

```diff
@@ -0,0 +1,140 @@
+package uk.gov.justice.digital.hmpps.historicalprisonerapi.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
+import java.time.LocalDate
+
+@JsonInclude(NON_NULL)
+data class PrisonerDetailDto(
```

Comment:
none of this really needed mapping, but better for the API to return the types etc. as frontend needs it anyway